tools/provision/formula/librdkafka.rb

```diff
@@ -0,0 +1,50 @@
+require File.expand_path("../Abstract/abstract-osquery-formula", __FILE__)
+
+class Librdkafka < AbstractOsqueryFormula
+  desc "The Apache Kafka C/C++ library"
+  homepage "https://github.com/edenhill/librdkafka"
+  url "https://github.com/edenhill/librdkafka/archive/v0.9.5.tar.gz"
+  sha256 "dd395ffca89c9591e567366f3ad2517cee76578a10d0a16a93f990c33f553179"
+
+  depends_on "openssl"
+  depends_on "pkg-config" => :build
+  depends_on "lzlib"
+  depends_on "lz4" => :recommended
```

Comment:
How important is `lz4` for Linux (since it's disabled on macOS)? Should we disable it completely on both?

Reply:
ATM, not important, since we're not currently using and have not even exposed that in the configuration yet.  In the future, seeing that lz4 does have some pretty awesome compression properties, it would definitely make since to have add this;  however, there's no absolutely no need for this now and if consistency between the OS's is a priority, than I think we should disable it.